project-base/storefront/components/Basic/Icon/GoogleMapMarkerIcon.tsx

```diff
@@ -0,0 +1,5 @@
+export const GoogleMapMarkerIcon: SvgFC = (props) => (
+    <svg {...props} viewBox="0 0 30 38" xmlns="http://www.w3.org/2000/svg">
+        <path d="M30,15A15,15,0,1,0,10.089,29.161L15,38l4.911-8.839A14.994,14.994,0,0,0,30,15Z" fill="currentColor" />
+    </svg>
+);
```

Comment:
What about `MarkerIcon`? Cannot that be used instead?

Reply:
The `MarkerIcon` looks different than the one we are using now for the SeznamMaps. I have tried it, but it is used in the stores section (screenshot below). It looked weird when used on the map.
![image](https://github.com/user-attachments/assets/91b89c77-2dae-4061-93d7-e5f5cf0b9539)